gtfs-realtime/spec/en/reference.md

```diff
@@ -502,11 +503,32 @@ An internationalized message containing per-language versions of a snippet of te
 
 A localized string mapped to a language.
 
+| _**Field Name**_ | _**Type**_ | _**Required**_ | _**Cardinality**_ | _**Description**_ |
+|------------------|------------|----------------|-------------------|-------------------|
+| **text** | [string](https://developers.google.com/protocol-buffers/docs/proto#scalar) | Required | One | A UTF-8 string containing the message. |
+| **language** | [string](https://developers.google.com/protocol-buffers/docs/proto#scalar) | Conditionally required | One | BCP-47 language code. Can be omitted if the language is unknown or if no internationalization is done at all for the feed. At most one translation is allowed to have an unspecified language tag - if there is more than one translation, the language must be provided. |
+
+## _message_ TranslatedImage
+
+An internationalized message containing per-language versions of an image. One of the images from a message will be picked up. The resolution proceeds as follows: If the UI language matches the language code of a translation, the first matching translation is picked. If a default UI language (e.g., English) matches the language code of a translation, the first matching translation is picked. If some translation has an unspecified language code, that translation is picked.
+
+**Caution:** this message is still **experimental**, and subject to change. It may be formally adopted in the future.
+
 #### Fields
 
 | _**Field Name**_ | _**Type**_ | _**Required**_ | _**Cardinality**_ | _**Description**_ |
 |------------------|------------|----------------|-------------------|-------------------|
-| **text** | [string](https://developers.google.com/protocol-buffers/docs/proto#scalar) | Required | One | A UTF-8 string containing the message. |
+| **localized_image** | [LocalizedImage](#message-localizedimage) | Required | Many | At least one localized image must be provided. |
+
+## _message_ LocalizedImage
+
+A localized image URL mapped to a language.
+
+| _**Field Name**_ | _**Type**_ | _**Required**_ | _**Cardinality**_ | _**Description**_ |
+|------------------|------------|----------------|-------------------|-------------------|
+| **url** | [string](https://developers.google.com/protocol-buffers/docs/proto#scalar) | Required | One | String containing a fully qualified URL linking to an image. The image linked needs to be less than 500kB. |
```

Comment:
Sorry for the late comment, I didn't think of this until now - I wonder if we should strongly encourage the producer to support (and consumer to use) [If-Modified-Since HTTP headers](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/If-Modified-Since) for this URL.

I could see producers swapping out the image at the URL for incidents as they develop and are more fully understood, and we really don't want people downloading a .5 MB file repeatedly just to see if it changed.

I suppose this applies to anything hosted at URLs (including the feeds themselves), but given the potentially large-ish file size here it seems more relevant.